plugins/webp-uploads/tests/test-uninstall.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Tests for webp-uploads plugin uninstall.php.
+ *
+ * @package webp-uploads
+ */
+
+class Test_WebP_Uploads_Uninstall extends WP_UnitTestCase {
+
+	/**
+	 * Test uninstall on a single site.
+	 */
+	public function test_uninstall_single_site(): void {
```

Comment:
While the above change will solve the issue of the report not generating, there's still a potential problem with multisite tests. Since the single site test will also run and `uninstall.php` is included using `include_once`, it won't execute again for multisite tests, causing those assertions to fail.

To prevent this issue, you might want to add this check to the single site uninstall test:
```suggestion
	public function test_uninstall_single_site(): void {
		if ( is_multisite() ) {
			$this->markTestSkipped( 'This test is for single site only.' );
		}

```

Reply:
Hi @ShyamGadde thank you so much for pointing me in the right direction.

> I'm curious about how you were able to generate the coverage reports locally.

I suppose I was just testing for multisite in my local env.
`npm run test-php-multisite:auto-sizes  -- -- -- --coverage-html=./html-reports` & probably that's why in my local the code coverage was showing up.

Thanks I'll make the necessary changes soon 🙇🏻